src/main/java/org/jenkinsci/test/acceptance/docker/Docker.java

```diff
@@ -65,6 +66,26 @@ public boolean isAvailable() {
     }
 
     /**
+     * Checks if a given container is currently running.
+     *
+     * @param container Container id
+     * @return false if the container is not running, true otherwise
+     */
+    public boolean isContainerRunning(String container) throws IOException, InterruptedException {
+        ProcessBuilder pBuilder = cmd("ps").add("-q", "--filter", "\"id=" + container + '"').build();
+        Process psProcess = pBuilder.start();
+        psProcess.waitFor();
```

Comment:
Damn mobile view!
There should only be a certain amount of output, but stream buffers are os depenant and writing could block causing the process to never exit.